### PR TITLE
Umbetov-Asman-Task-1

### PR DESCRIPTION
**Pull request by fastndead**

### CODE REVIEW

File: mock1/public/css/style.css [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+}
+
+.container__icon{
+  width:55px;
+  height:55px;
+  margin-top: 75px;
+}
+
+.container_tiles{
+    margin-top: 0;
```
Comment (vadeneev) [inline mock1/public/css/style.css:75]: please fix all indents

File: mock1/public/css/style.css [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+  background-position: center;
+  background-repeat: no-repeat;
+  font-size: 1rem;
+  text-decoration: none;
+  letter-spacing: 5.4px;
+  margin-bottom: 6px;
+	transition: .1s;
+}
+
+.tile > span{
```
Comment (vadeneev) [inline mock1/public/css/style.css]: It is not about BEM, right ?

**Review by vadeneev — CHANGES_REQUESTED** [review]

File: mock1/public/css/style.css [hunk 3]
```diff
@@ -0,0 +1,264 @@
+html{
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body{
```
Comment (vovcyan) [inline mock1/public/css/style.css]: Please format this file

File: mock1/public/css/style.css [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+}
+.wrapper{
+  display: flex;
+  width: 100vw;
+  align-content: center;
+  justify-content: center;
+}
+
+.wrapper_header{
+  background: url(../images/header-background.png) center;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: background-image

File: mock1/public/css/style.css [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+  background-size: cover;
+  background-position: top right;
+
+  min-height: 100vh;
+}
+
+.wrapper_midpage-with-background{
+  background: url(../images/midpage-background.png) center;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: background-image: ...
background-position: center

File: mock1/public/css/style.css [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 80px;
+}
+
+.container_header{
+  margin-top: 27vh;
+  margin-bottom: 0;
+
```
Comment (vovcyan) [inline mock1/public/css/style.css:50]: Remove it

File: mock1/public/css/style.css [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+  margin-top: 0;
+  margin-bottom: 0;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100vw;
+}
+
+.container_bordered-white{
+  border: 2px solid white;
+  border-top: 0px;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: without px

File: mock1/public/css/style.css [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+}
+
+.container__tiles-row{
+  width:100vw;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: Format it

File: mock1/public/css/style.css [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+  justify-content: center;
+}
+
+.text_paragraph{
+  width: 62%;
+  margin-top: 29px;
+  font-size: 10px;
+  font-weight: 300;
+  line-height: 16px;
+  letter-spacing: 0.6px;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: .6px

File: mock1/public/css/style.css [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+  letter-spacing: 5.75px;
+  font-weight: 700;
+}
+
+.text_heading_paragraph{
+  font-weight: 400;
+  width: 100%;
+  line-height: 19.33px;
+  text-transform: uppercase;
+  margin-left: 0px;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: 0

File: mock1/public/css/style.css [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+  flex-grow: 1;
+  flex-basis: 100px;
+  height: 158px;
+  background-position: center;
+  background-repeat: no-repeat;
+  font-size: 1rem;
+  text-decoration: none;
+  letter-spacing: 5.4px;
+  margin-bottom: 6px;
+	transition: .1s;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: Remove spaces

File: mock1/public/css/style.css [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+.tile_center{
+  max-width: 802px;
+  flex-grow: 2;  
+  height: 158px;
+  margin-right: 7px;
+  margin-left: 7px;
+}
+
+.tile:hover{
+  opacity: 0.5;
```
Comment (vovcyan) [inline mock1/public/css/style.css]: .5

File: mock1/public/css/style.css [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+  margin-right: 170px;
+  font-size: 8px;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: 1.27px;
+  border: 1px solid #3c3c3c;
+  background-color: #3c3c3c;
+}
+
+.footer__logo{
```
Comment (vovcyan) [inline mock1/public/css/style.css]: Format it

File: mock1/public/css/style.css [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+@media (max-width: 1024px){
+  .text_heading_second {
+    font-size: 15px;
+  }
+}
```
Comment (vovcyan) [inline mock1/public/css/style.css:281]: Please add media for vertical iPad

File: mock1/public/index.html [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+            We help you to fill the gap between development,<br>
+            design and product management.
+          </p>
+          <a class="container__icon_bottom" href="#firstSection" >
+          <img class="container__icon_header container__icon_bottom" src="/public/images/mouse-logo.png" alt="mouse-logo">
+          </a>
+      </div>
+    </div>
+
+    <div class="wrapper" id="firstSection">
```
Comment (vovcyan) [inline mock1/public/index.html:32]: Please do not use id at all

File: mock1/public/index.html [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+          We start each project by understanding your business goals, 
+          the user’s needs and the product requirements. 
+          Then we help translate them into effective and elegant solutions.
+        </p>
+      </div>
+    </div>
+
+    <div class="wrapper wrapper_midpage-with-background">
+      <div class="container container_bordered container_bordered-white">
+        <h2 class="text text_white text_heading_second  text_heading_second_white"><span>Services</span></h2>
```
Comment (vovcyan) [inline mock1/public/index.html]: Please separate it into 2 lines

File: mock1/public/index.html [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+        Before we sign a contract or write a line of code our team will spend the necessary time needed to understand your product vision. <br>
+        The outcome of this scoping session is a proposal or high-level statement of work. 
+          Based on your requirements, we can provide a complete offering or just the pieces you need.  
+        </p>
+      </div>
+    </div>
+    
+    <div class="wrapper">
+      <div class="container container_bordered">
+        <h2 class="text text_heading_second"><span>What we do</span></h2>
```
Comment (vovcyan) [inline mock1/public/index.html]: The same

File: mock1/public/index.html [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+         <img class="container__icon" src="/public/images/design-icon.png" alt="design-icon">
```
Comment (vovcyan) [inline mock1/public/index.html]: Please change alt attribute

File: mock1/public/index.html [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+         <h3 class="text_heading_third">Design</h3>
+         <p class="text text_paragraph">All engineers are fluent in the latest enterprise, mobile and web development technologies. 
+        They collaborate with your team to write, test, and improve code on a daily basis, using proven practices such as test-driven development and pair programing.
+        </p>
+         <img class="container__icon" src="/public/images/development-icon.png" alt="development-icon">
```
Comment (vovcyan) [inline mock1/public/index.html]: The same

File: mock1/public/index.html [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+         <h3 class="text_heading_third">Development</h3>
+         <p class="text text_paragraph">
+           Our design practice offers a full range of services including branding strategy,
+ interaction and visual design and user experience testing.
```
Comment (vovcyan) [inline mock1/public/index.html]: Please format text

File: mock1/public/index.html [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+          Planning and development is iterative. Because we are constantly coding and testing, the products we build are always ready to go live. This iterative process allows for changes as business requirements evolve. 
+        </p>
+      </div>
+    </div>
+    
+    <div class="wrapper">
+      <div class="container container_tiles">
+        <div class="container__tiles-row">
+           <a class="tile tile-1 text_white" href="#">
+            <span class="tile__label">topworkz</span>
```
Comment (vovcyan) [inline mock1/public/index.html]: Why 3 spaces there?

File: mock1/public/index.html [hunk 22]
```diff
@@ -0,0 +1,137 @@
...
+          <a class="tile tile-5 tile_center text_white" href="#">
+              <span class="tile__label">lable</span> 
+          </a>
+
+          <a class="tile tile-6 text_white" href="#">
+              <span class="tile__label">kyiv</span>  
+          </a>
+        </div>
+       
+
```
Comment (vovcyan) [inline mock1/public/index.html]: Remove it

File: mock1/public/index.html [hunk 23]
```diff
@@ -0,0 +1,137 @@
...
+       
+
+      </div>
+    </div>
+
+    <div class="wrapper">
+      <div class="container container_bordered">
+        <h2 class="text text_heading_second"><span>Contact us</span></h2>
```
Comment (vovcyan) [inline mock1/public/index.html]: The same

File: mock1/public/index.html [hunk 24]
```diff
@@ -0,0 +1,137 @@
...
+
+    <footer class="wrapper">
+      <div class="container">
+      <img class="footer__logo" src="/public/images/footer-logo.png" alt="footer-logo">
+      <p class="text_footer">
+        © 2014. All Rights Reserved Ellipsis.com
+      </p>
+      </div>
+    </footer>
+
```
Comment (vovcyan) [inline mock1/public/index.html]: Remove it

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: mock1/public/css/style.css [hunk 25]
```diff
@@ -0,0 +1,279 @@
...
+.text_heading_second {
+    font-size: 23px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 8.95px;
+    padding-left: 77px;
+    padding-right: 77px;
+    display: table;
+    width: 100.5%;
+    ;
```
Comment (vadeneev) [inline mock1/public/css/style.css]: 🐘

File: mock1/public/css/style.css [hunk 26]
```diff
@@ -0,0 +1,279 @@
...
+
+.text_heading_second {
+    font-size: 23px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 8.95px;
+    padding-left: 77px;
+    padding-right: 77px;
+    display: table;
+    width: 100.5%;
```
Comment (vadeneev) [inline mock1/public/css/style.css]: so weird, is that required ?

File: mock1/public/css/style.css [hunk 27]
```diff
@@ -0,0 +1,279 @@
...
+    font-size: 13px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 3.51px;
+}
+
+.text_heading_second:before,
+.text_heading_second>.text_heading-inner-text,
+.text_heading_second:after {
+    display: table-cell;
```
Comment (vadeneev) [inline mock1/public/css/style.css]: really ? Why do we need that ?

File: mock1/public/css/style.css [hunk 28]
```diff
@@ -0,0 +1,279 @@
...
+    background-color: #3c3c3c;
+}
+
+.footer__logo {
+    width: 18px;
+    height: 19px;
+    margin-right: 18px
+}
+
+footer>.container {
```
Comment (vadeneev) [inline mock1/public/css/style.css]: It is not allowed to use element selector in BEM

**Review by vadeneev — CHANGES_REQUESTED** [review]

**Review by vadeneev — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**